bcipy/parameters/parameters.json

```diff
@@ -876,5 +876,13 @@
     "helpTip": "What should the name of the main log file be for each BciPy session",
     "recommended_values": "",
     "type": "str"
+  }, 
+  "section_names": {
+    "value": "{'aaaa_booleans': 'Important BciPy Parameters', 'acq_config': 'ACQ Configuration', 'bci_config': 'BCI Configuration', 'feedback_config': 'Feedback Configuration', 'lang_model_config': 'Language Model Configuration', 'signal_config': 'Signal Configuration'}",
+    "section": "bci_config",
+    "readableName": "Names of parameter sections",
+    "helpTip": "Names of parameter sections",
+    "recommended_values": "",
+    "type": "str"
```

Comment:
This is not a string, let's call it an object and modify the cast values function to support his type